app/views/pages/notif.html.erb

```diff
@@ -0,0 +1,30 @@
+<head>
+<script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" async=""></script>
+<script>
+  var OneSignal = window.OneSignal || [];
+  OneSignal.push(function() {
+    OneSignal.init({
+      appId: "b290fd9a-eedf-44b0-8bfd-6a37646957b6",
+    });
+  });
+
+  let netid = '<%= current_user.netid %>'; 
+  console.log(netid)
```

Comment:
Will take out the console.log, but otherwise since I have yet to do it via React, I will remove it and the route from routes.rb once we have it fully working via React, as otherwise there is no other way to accept receiving notifications.

Reply:
sounds fair, if you push your changes, it will resolve them on github btw